sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,14 +56,27 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit ec634ff05b067d7922ec45059dda94665e5dcd9b
-  # Merge: 418ad535 8714b853
-  # Author: Piotr Balcer <piotr.balcer@intel.com>
-  # Date:   Thu Mar 14 15:52:52 2024 +0100
-  #     Merge pull request #1438 from PatKamin/disable-fuzztests
-  #     Disable fuzz tests on ubuntu-22.04 runner
-  set(UNIFIED_RUNTIME_TAG ec634ff05b067d7922ec45059dda94665e5dcd9b)
+  #set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/JackAKirk/unified-runtime.git")
+
+  # commit 51d7180c344bbc2f942533e5fc51b0b04871f8d5
+  # Merge: b66cf9b1 0e37380e
+  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
+  # Date:   Fri Jan 26 12:20:20 2024 +0000
+  #     Merge pull request #1205 from ykhatav/ur_dependentload
+  #     [UR] add dependent-load flag to exclude CWD from default search path …
+  set(UNIFIED_RUNTIME_TAG 35b6cd09dd92b04eae8195a706ef6c63541b3cf1)
+#=======
+
+#  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime")
+  # commit 3225b822b5d8cbfa85d7fc1bd5a5bf96e5bb8c1a
+  # Merge: edb281f3 5fc41099
+  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
+  # Date:   Tue Jan 30 12:31:44 2024 +0000
+  #     Merge pull request #1168 from Seanst98/sean/unique-addr-mode-per-dim-adapters
+  #     [Bindless][CUDA] Unique addressing modes per dimension
+#  set(UNIFIED_RUNTIME_TAG 3225b822b5d8cbfa85d7fc1bd5a5bf96e5bb8c1a)
+
```

Comment:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 29ee45c4451a682f744146cc9dbeb2617ecdd6b3
  # Merge: db4b0c14 4f5d005a
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Mon Mar 18 12:14:26 2024 +0000
  #     Merge pull request #1291 from JackAKirk/cuda-seq-cst-b
  #     [CUDA] Report that devices with cc >= sm_70 support seq_cst
  set(UNIFIED_RUNTIME_TAG 29ee45c4451a682f744146cc9dbeb2617ecdd6b3)
```

Reply:
I don't see the UR repo/tag change in this PR

Reply:
Sorry I misunderstood; is it correct now?

Reply:
OK fixed that now thanks.